system/views/general/_openGraphMeta.cfm

```diff
@@ -1,16 +1,18 @@
 <cfscript>
-	local.teaser       = Trim( event.getPageProperty( "teaser"        ) );
-	local.description  = Trim( event.getPageProperty( "description"   ) );
-	local.browsertitle = Trim( event.getPageProperty( "browser_title" ) );
-	local.title        = Trim( event.getPageProperty( "title"         ) );
-	local.mainImage    = Trim( event.getPageProperty( "main_image"    ) );
+	local.teaser       = Trim( event.getPageProperty( "teaser"             ) );
+	local.description  = Trim( event.getPageProperty( "description"        ) );
+	local.browsertitle = Trim( event.getPageProperty( "browser_title"      ) );
+	local.title        = Trim( event.getPageProperty( "title"              ) );
+	local.mainImage    = Trim( event.getPageProperty( "main_image"         ) );
+	local.ogType       = Trim( event.getPageProperty( "og_type", "website" ) );
```

Comment:
I think the ogType should probably belong on the `site` record (if you were to change it). So:

```
local.site = event.getSite():
local.ogType = local.site.og_type ?: "website";
```